packages/init/src/steps/starters/s3.ts

```diff
@@ -0,0 +1,187 @@
+import fs from "fs";
+import axios from "axios";
+import mime from "mime";
+import snakeCase from "lodash.snakecase";
+import path from "path";
+import FormData from "form-data";
+import uniqid from "uniqid";
+import { logs } from "../../utils";
+import {
+  Component,
+  ComponentInfo,
+  VariationSM,
+  SliceSM,
+} from "@slicemachine/core/build/models";
+
+export type ALC = {
+  values: {
+    url: string;
+    fields: Record<string, string>;
+  };
+  imgixEndpoint: string;
+  err: null | string;
+};
+
+export async function createAcl(
```

Comment:
That's really cool, because we will be able to just delete that part and replace it